docker/node-redis.Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM redis:5.0 as redis
 
-FROM node:10-buster
+FROM node:12.16.3-buster
```

Comment:
I would better use `node:12-buster` to just stick to the latest version (without having too much risk of something breaking because an external cause).